apps/lockfile-explorer-web/src/helpers/displaySpecChanges.ts

```diff
@@ -0,0 +1,14 @@
+import { ISpecChange } from '../parsing/compareSpec';
+
+export const displaySpecChanges = (specChanges: Map<string, ISpecChange>, dep: string): string => {
```

Comment:
In the [Adobe XD mockup](https://xd.adobe.com/view/13b05d87-8967-4485-b11a-fdb0e4fdf93d-46e3/?fullscreen&hints=off), I used a fixed-width font to make columns:

![image](https://user-images.githubusercontent.com/4673363/202326731-93099049-ff64-473f-8602-2174850bc44a.png)

I like your approach of using colors:
![image](https://user-images.githubusercontent.com/4673363/202327065-5737a15f-530c-4323-a696-e8e979f399b6.png)
...but maybe there is some way to line up the versions so they are easier to read?

Reply:
fixed!